recipe/meta.yaml

```diff
@@ -10,12 +10,11 @@ package:
   version: {{ version }}
 
 source:
-  fn: scorec_after_sim.tar.gz
-  url: https://github.com/SCOREC/core/archive/after_sim.tar.gz
+  url: https://github.com/SCOREC/core/archive/v2.2.2.tar.gz
   sha256: {{ sha256 }}
-  patches:
-    - 0001-fix-malloc-usage.patch  # [osx]
-    - 0002-fmemopen-memstream-ports-for-darwin.patch  # [osx]
+#  patches:
+#    - 0001-fix-malloc-usage.patch  # [osx]
+#    - 0002-fmemopen-memstream-ports-for-darwin.patch  # [osx]
```

Comment:
What about the OSX patches, aren't them needed anymore?

Reply:
When they were turned on previously, the osx-related tests on the Azure pipeline were failing. Turning the patches off seems to allow the tests to pass. Is there a more reliable way of checking whether the OSX patches are necessary?

Reply:
I'm not sure, I guess we can leave it like that and we will look at it if we have issues.